core/src/main/scala/org/bitcoins/core/protocol/script/ScriptWitness.scala

```diff
@@ -94,6 +99,13 @@ sealed abstract class P2WSHWitnessV0 extends ScriptWitnessV0 {
     RawScriptPubKey.fromBytes(cmpct.bytes ++ stack.head)
   }
 
+  lazy val pubKeys: Seq[ECPublicKey] = {
+    val pubKeys: Seq[ScriptToken] = redeemScript.asm
+      .filter(_.isInstanceOf[ScriptConstant])
+      .filterNot(_.isInstanceOf[ScriptNumberOperation])
+    pubKeys.map(k => ECPublicKey(k.hex))
+  }
```

Comment:
This seems unsafe, not all script constants are going to be public keys (hashes for example), you're going to want to check `ECPublicKey.isFullyValid` as well